docs/buildtriage.md

```diff
@@ -0,0 +1,46 @@
+## Build Triage Rotation
+
+The responsibility of this role is triaging our rolling / official builds, filing issues to track broken tests, submitting changes to dotnet/runtime to work around issues (disabling a test, undoing a PR that broke the build). 
+
+In some cases this will require working with core-eng team when the issues are in Helix / Azure / Arcade. This person will also attend CI Council with the infra manager to provide updates on our reliability status.
+
+This directly impacts developer productivity and the need to promptly fix such breaks can span across time zones. Hence it will be the collective responsibility of the Scout pool to investigate such breaks. 
+
+This role will work on a rotation basis. There are six people in the role and each rotation will last for a calendar month.
+
+## Tracking Build Failures
+All the CI failures can be tracked through the CI Council dashboards i.e.  [Public](https://dev.azure.com/dnceng/public/_dashboards/dashboard/40ac4990-3498-4b3a-85dd-2ffde961d672), [Internal](https://dev.azure.com/dnceng/internal/_dashboards/dashboard/e1bb572d-a2b0-488f-a58a-54c73a547f0d).
+We have different dashboards for public(Rolling & PR Builds) and internal builds. 
+
+Tests are not run during the internal builds. Publishing and signing steps are run only during  internal builds. Rolling builds runs tests for the full matrix. 
+
+Contact @chcosta if you are having any trouble accessing the dashboards.
+
+## Ongoing Issues
+
+All the issues causing the builds to fail should be marked with ```blocking-clean-ci``` label.
+
+It helps in tracking issues effectively. There are some issues that are reproduced more often than others and should be fixed with a higher priority in the next rotation. Such issues are mentioned in the following table.
+
+| Issue | Comments  | 
```

Comment:
Should this list be rather tracked as a github project? We typically do not check in (frequently changing) project status into the repo.

Reply:
The main purpose i added this was to have a shortlist of issues for the new person (which will be every month) to give more priority to. We will update the doc every month with pass rates. i think we can keep this for couple of months and remove if its not helpful or a hassle to maintain.
In any case i think we should have a project board with all the blocking-clean-ci label.

Reply:
The churn in high-priority CI issues is very high. It is changing literally every day. I believe that the list you are adding is largely outdated:

- https://github.com/dotnet/runtime/issues/35101 has no data about it being actual CI affecting problem. Is it a real problem for CI stability? I think it should be just closed. The original data in the issue were collected on debug build of the runtime that is slow by design and not actually run in the CI. 
- https://github.com/dotnet/runtime/issues/41078 is fixed in master already (test disabled against active issue). Just waiting for release/5.0 port.
- https://github.com/dotnet/runtime/issues/35916 cannot be affecting CI. The tests that it talks about are disabled already.